src/main/java/org/jabref/gui/maintable/MainTableHeaderContextMenu.java

```diff
@@ -224,4 +234,20 @@ public boolean isVisibleInTable() {
             return visibleInTable;
         }
     }
+
+    /**
+     * RightClickMenuMoreItem: A special menu item that opens the Preferences > Entry table window
+     *
+     */
+    private class RightClickMenuMoreItem extends MenuItem {
+
+        RightClickMenuMoreItem(String displayName) {
+            super(displayName);
+
+            // Set action to open the Preferences > Entry table window
+            this.setOnAction(event -> {
+                new ShowPreferencesAction(JabRefGUI.getMainFrame(), Globals.TASK_EXECUTOR, new TableTab()).execute();
```

Comment:
Please do not use JabRefGui, Globals and JabRefFrame. Aim for loose coupling.

Reply:
Would it be a good thing to provide these parameters at the constructor - or would you rather go for a lambda?

Reply:
Unfortunately, I am not sure how to resolve this.

Reply:
![image](https://user-images.githubusercontent.com/1366654/235272262-d8b7f1bd-6209-4498-999b-c532e7fc8564.png)